### PR TITLE
Add coq-smlms 1.0.0

### PR DESCRIPTION
**Pull request by hferee**

### CODE REVIEW

File: released/packages/coq-smlms/coq-smlms.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+license: "MIT"
+build: [
+  [make "-j%{jobs}%"]
+  [make "html"] {with-doc}
+]
+install: [make "install"]
+remove: [
+  [make "uninstall"]
+  ["rmdir" "%{lib}%/coq/user-contrib/AbstractRenaming"]
+]
```
Comment (palmskog) [inline released/packages/coq-smlms/coq-smlms.1.0.0/opam:20]: Removal is supposed to be handled automatically by opam these days, especially when using coq_makefile.
```suggestion
```

File: released/packages/coq-smlms/coq-smlms.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+name: "coq-smlms"
```
Comment (palmskog) [inline released/packages/coq-smlms/coq-smlms.1.0.0/opam:2]:
```suggestion
```
The name is implicitly derived from the file path.

File: released/packages/coq-smlms/coq-smlms.1.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  [make "-j%{jobs}%"]
+  [make "html"] {with-doc}
+]
+install: [make "install"]
+depends: [
+  "ocaml"
+  "coq" {>= "8.15.1"}
+]
+tags: [
+  "category:CS/Semantics and Compilation/Semantics"
```
Comment (silene) [inline released/packages/coq-smlms/coq-smlms.1.0.0/opam:21]:
```suggestion
  "category:Computer Science/Semantics and Compilation/Semantics"
```

File: released/packages/coq-smlms/coq-smlms.1.0.0/opam [hunk 4]
```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+version: "1.0.0"
```
Comment (palmskog) [inline released/packages/coq-smlms/coq-smlms.1.0.0/opam:2]:
```suggestion
```

**Pull request merged**